### ISSUE TITLE
Show individual discussions in a split window

### ISSUE THREAD

**Issue by jakubbortlik** [harrisoncramer/gitlab.nvim#141]
## Feature Description

In #126 I suggest to implement filtering of discussions. After some time of using `gitlab.nvim` I think that a more useful feature would be showing discussion content separately from the discussions list. I imagine it could look something like this:
![image](https://github.com/harrisoncramer/gitlab.nvim/assets/17056143/a81bb2bf-ca40-4883-8e06-29ec96f992d7)
![image](https://github.com/harrisoncramer/gitlab.nvim/assets/17056143/cafb2382-d2b1-4c5d-9be5-1e4947945ca5)

I think this would make navigation in discussions much easier - now I very often lose track what thread a specific comment belongs to. I would very much prefer having this implemented before any filtering.

**Comment by harrisoncramer** [comment]
Hello Jakub,

With this plugin users can have the tree underneath the reviewer; to the side of the reviewer; or even underneath/side of a normal file (outside of the reviewer). Implementing these views as a split would probably interfere with most workflows so it's not something I'm keen to merge. However, someone could implement a preview _popup_ which would be very similar to what you have here. 

Essentially replicating the current contents of a discussion into a smaller, fully expanded subtree like you've shown here, and then render that in a popup view. It'd essentially involve duplicating the contents of the highlighted discussion into the popup buffer. The popup would have to be positioned relative to the current split discussion tree. Definitely worth considering, but it'd be a pretty hefty feature add. Thank you for the idea!

**Comment by jakubbortlik** [comment]
Hi Harrison,

Thanks for your reply, let me add some more detail on why I came up with the idea. When I first installed `gitlab.nvim`, the discussions were **already** a split window - discussions and notes. Even after they have been merged into a single window, I still prefer to have them below the reviewer, so that the reviewer windows are not too narrow (leading to wrapped lines etc.). But now I've noticed that a considerable part of my screen is empty - the place where there used to be notes.

Placing individual discussions into that empty spot seemed like a double win - using otherwise wasted space and also making the discussions more easily navigable - currently it's really difficult for me to see what are threads in a single discussions and where are the boundaries between discussions.

Maybe the clearer separation of individual discussions could be helped by adding some kind of indent markers (lines that connect nodes on the same level) in the tree. Otherwise I'll probably just stick to the `simple` tree type.
